### ISSUE TITLE
Accept option for owner of commit

### ISSUE THREAD

**Issue by PatrykKaczmarek** [netguru/review#33]
Hey, I discovered something which isn't consistent with previous version of review app:

<img width="1460" alt="screen shot 2015-08-04 at 12 58 05" src="https://cloud.githubusercontent.com/assets/3520281/9059263/bf3eb528-3aa8-11e5-84ab-0b2970d034c7.png">

Although `Accept` does nothing, it is confusing to see it there. Any reason stays behind it?

**Comment by mamut** [comment]
agreed, it should be obvious that you can't do it, by showing this option greyed out or not at all

**Comment by nekath** [comment]
+1 for greyed out
